plugin/pkg/scheduler/metrics/metrics.go

```diff
@@ -68,3 +80,17 @@ func Register() {
 func SinceInMicroseconds(start time.Time) float64 {
 	return float64(time.Since(start).Nanoseconds() / time.Microsecond.Nanoseconds())
 }
+
+// ReportBindingRateLimiterSaturation exports status of binding rate limiter
+// whether it's full (1) or not (0).
+func ReportBindingRateLimiterSaturation(rl util.RateLimiter) {
+	for {
+		if rl.CanAccept() {
+			BindingRateLimiterSturation.Set(0)
```

Comment:
I don't have strong opinion on the api, but we definitely should export the saturation, not just 0 or 1 (which isn't very informative).